roles/telemetry_autoscaling/tasks/test_services.yml

```
@@ -17,14 +17,38 @@
   failed_when:
     - result.stdout != "True"
 
-# - name: Verify that custom resource AUTOSCALING is ready
-# It is returning an unexpected response and needs further debugging. will raise another PR to inspect this
-#   ansible.builtin.command:
-#     cmd: |
-#       oc get autoscaling autoscaling -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}{"\n"}'
-#   register: result
-#   failed_when:
-#     - result.stdout != "True"
+#It is returning an unexpected response and needs further debugging
+- name: Enclose this task in a block/rescue to inspect further
+  block:
+    - name: Verify that custom resource AUTOSCALING is ready
+      ansible.builtin.command:
+        cmd: |
+          oc get autoscaling autoscaling -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}{"\n"}'
+      register: result
+      failed_when:
+        - result.stdout != "True"
+  rescue:
+    - name: view autoscaling objects
+      ansible.builtin.command:
+        cmd: |
+          oc get autoscaling -oyaml
```

Technically, you can run both commands here, on two lines.

```suggestion
          oc get autoscaling -oyaml
          oc get telemetry telemetry -oyaml
      ignore_errors: true
```
Then you only have to show one set of results.
I would recommend ignoring errors, for this so that the output is always shown, since errors might be useful as well.